shared/src/packages/drivers/libs/validation-schemas/driver-create-update-request-body.validation-schema.ts

```diff
@@ -6,19 +6,21 @@ import { UserValidationRule } from '~/packages/users/libs/validation-schemas/enu
 import { DriverValidationMessage } from '../enums/enums.js';
 import { type DriverCreateUpdateRequestDto } from '../types/types.js';
 
+const driverLicenseNumber = joi
+  .string()
+  .trim()
+  .regex(UserValidationRule.DRIVER_LICENSE_NUMBER)
+  .required()
+  .messages({
+    'string.empty': DriverValidationMessage.DRIVER_LICENSE_NUMBER_REQUIRED,
+  });
+
 const driverCreateUpdateRequestBody = joi.object<
   DriverCreateUpdateRequestDto,
   true
 >({
   ...commonSignUpRules,
-  driverLicenseNumber: joi
-    .string()
-    .trim()
-    .regex(UserValidationRule.DRIVER_LICENSE_NUMBER)
-    .required()
-    .messages({
-      'string.empty': DriverValidationMessage.DRIVER_LICENSE_NUMBER_REQUIRED,
-    }),
+  driverLicenseNumber,
 });
 
-export { driverCreateUpdateRequestBody };
+export { driverCreateUpdateRequestBody, driverLicenseNumber };
```

Comment:
Perhaps now it makes sense to rename this validation scheme to driverUpdateRequestBody. After all, it is no longer relevant for creating a driver, and the same applies to the DriverCreateUpdateRequestDto type, rename it to DriverUpdateRequestDto.